assets/js/app.js

```diff
@@ -10,6 +10,14 @@ import css from "../css/app.scss";
 // Import dependencies
 //
 import "phoenix_html";
+import LiveSocket from "phoenix_live_view";
+
+// LiveView polyfills for IE11
+import "mdn-polyfills/NodeList.prototype.forEach";
+import "mdn-polyfills/Element.prototype.closest";
```

Comment:
Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)